.github/workflows/docker-publish.yml

```
@@ -0,0 +1,117 @@
+name: 🐳 Build and Push Docker Image
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)